x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -805,34 +784,49 @@ export const getAllCasesStatuses = async (
   const { body: statuses } = await supertest
     .get(CASE_STATUS_URL)
     .set('kbn-xsrf', 'true')
-    .send()
     .expect(expectedHttpCode);
 
   return statuses;
 };
 
-export const getCase = async (
-  supertest: st.SuperTest<supertestAsPromised.Test>,
-  caseId: string,
-  includeComments: boolean = false,
-  expectedHttpCode: number = 200
-): Promise<CaseResponse> => {
+export const getCase = async ({
+  supertest,
+  caseId,
+  includeComments = false,
+  expectedHttpCode = 200,
+  auth = { user: superUser, space: null },
+}: {
+  supertest: st.SuperTest<supertestAsPromised.Test>;
+  caseId: string;
+  includeComments?: boolean;
+  expectedHttpCode?: number;
+  auth?: { user: User; space: string | null };
```

Comment:
I did like this because I wanted to produce a TS error when when someone provides a user but not a space. This error will make them think why they don't want to provide a space and take a decision about it.

Reply:
Ah gotcha. That makes sense. If we run into a situation where we want to pass in the user but use the default space we can change it then 👍